.github/ISSUE_TEMPLATE/other-issue.md

```diff
@@ -0,0 +1,6 @@
+---
+name: How-to and other issues
+about: Please, post other questions on StackOverflow with `gpflow` tag.
+---
+
+**For questions on how to work with GPflow, or support for problems that are not verified bugs in GPflow, please go to [StackOverflow](https://stackoverflow.com/questions/tagged/tensorflow) and use the [gpflow tag](https://stackoverflow.com/questions/tagged/gpflow) for your question.** This makes it easier for other users of GPflow to help you with your problem, and also makes your question and any answers more easily discoverable for other users who may come across the same question in the future!
```

Comment:
don't we want users to post _all_ bugs here? I don't think SO is a good place for bug reports, partly cos it's not meant for raising bug reports (though it can be a place to suggest workarounds), and partly cos we'll just end up loudly publicising our history of bugs, which looks bad

Reply:
The motivation was to split between "how do I do this specific thing?" that are good for stackoverflow from "this thing is actually broken". I'd love to hear your suggestions how to improve the wording and make that clearer!

Reply:
(i.e., all questions that we would answer *without* changing the code or documentation should go on stackoverflow)